### ISSUE TITLE
Possible to pass env variables for external prompt's path?

### ISSUE THREAD

**Issue by albertlieyingadrian** [promptfoo/promptfoo#1685]
For example, I'd love to pass `env.ROOT_PATH = ${pwd}` here
```yaml
prompts:
  - file://{{ env.ROOT_PATH }}/something.txt

providers:
  - label: 'gpt 3.5 low temperature'
    id: openai:gpt-3.5-turbo
    config:
      temperature: 0.1
```

**Comment by albertlieyingadrian** [comment]
Here you go @mldangelo https://github.com/promptfoo/promptfoo/pull/1688

**Comment by mldangelo** [comment]
Hi @albertlieyingadrian, thank you so much for your contribution! This was just released in version [0.89.0](https://github.com/promptfoo/promptfoo/releases/tag/0.89.0). I appreciate your help in making promptfoo better. Please keep the issues and PRs coming.

**Comment by albertlieyingadrian** [comment]
Thanks! @mldangelo I also added a request for other config's keys as well https://github.com/promptfoo/promptfoo/issues/1700
